Bottom: 86f2d7fc57f930af664fb1aabe01f6ec9664fc83
Top:    0389aa25ffe6fa429d63a20ee5b08bec05948ba6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-05 01:14:04 +0100

Refresh of autoroute-c-when-live-drawing-

---

diff --git a/src/autoroute.c b/src/autoroute.c
index 1823b77..c766c14 100644
--- a/src/autoroute.c
+++ b/src/autoroute.c
@@ -3210,8 +3210,9 @@ RD_DrawVia (routedata_t * rd, LocationType X, LocationType Y,
   if (TEST_FLAG (LIVEROUTEFLAG, PCB))
     {
        live_via = CreateNewVia (PCB->Data, X, Y, radius * 2,
-                                2 * rb->style->Keepaway, 0,
-                                rb->style->Hole, NULL, MakeFlags (0));
+                                2 * AutoRouteParameters.style->Keepaway, 0,
+                                AutoRouteParameters.style->Hole, NULL,
+                                MakeFlags (0));
        if (live_via != NULL)
          DrawVia (live_via);
     }
@@ -3256,7 +3257,7 @@ RD_DrawVia (routedata_t * rd, LocationType X, LocationType Y,
       /* and add it to the r-tree! */
       r_insert_entry (rd->layergrouptree[rb->group], &rb->box, 1);
       rb->flags.homeless = 0;	/* not homeless anymore */
-      rb->livedraw_obj.via = via;
+      rb->livedraw_obj.via = live_via;
     }
 }
 static void
